package.json

```diff
@@ -18,20 +25,59 @@
   "author": "Grafana Labs",
   "license": "MIT",
   "devDependencies": {
-    "@grafana/data": "8.0.0",
-    "@grafana/runtime": "8.0.0",
-    "@grafana/toolkit": "8.0.0",
-    "@grafana/ui": "8.0.0",
-    "@types/lodash": "latest",
+    "@babel/core": "^7.16.7",
+    "@grafana/e2e": "9.1.2",
+    "@grafana/e2e-selectors": "9.1.2",
+    "@grafana/eslint-config": "^5.1.0",
+    "@grafana/tsconfig": "^1.2.0-rc1",
+    "@swc/core": "^1.2.144",
+    "@swc/helpers": "^0.4.12",
+    "@swc/jest": "^0.2.20",
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.4",
+    "@types/glob": "^8.0.0",
+    "@types/jest": "^29.2.2",
+    "@types/lodash": "^4.14.188",
+    "@types/node": "^18.11.9",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
+    "@typescript-eslint/parser": "^5.42.1",
+    "copy-webpack-plugin": "^11.0.0",
+    "cspell": "6.13.3",
+    "css-loader": "^6.7.1",
+    "eslint": "8.26.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-jsdoc": "^39.6.2",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.26.1",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-webpack-plugin": "^3.1.1",
+    "fork-ts-checker-webpack-plugin": "^7.2.0",
+    "glob": "^8.0.3",
+    "identity-obj-proxy": "3.0.0",
+    "jest": "27.5.0",
     "lodash": "latest",
     "moment": "2.29.2",
-    "tslib": "2.3.1"
+    "prettier": "^2.5.0",
+    "replace-in-file-webpack-plugin": "^1.0.6",
+    "sass": "1.56.1",
+    "sass-loader": "13.2.0",
+    "style-loader": "3.3.1",
+    "swc-loader": "^0.2.3",
+    "ts-node": "^10.5.0",
+    "tsconfig-paths": "^4.1.0",
+    "tslib": "2.3.1",
+    "typescript": "^4.4.0",
+    "webpack": "^5.69.1",
+    "webpack-cli": "^4.9.2",
+    "webpack-livereload-plugin": "^3.0.2"
   },
   "dependencies": {
-    "moment-timezone": "0.5.34"
-  },
-  "volta": {
-    "node": "14.17.6",
-    "yarn": "1.22.11"
+    "@emotion/css": "^11.1.3",
+    "@grafana/data": "9.1.2",
```

Comment:
Did you bump these `@grafana` dependencies or did the create-plugin migrate script do it?

Reply:
~Gangster rap made me do it~ `create-plugin migrate` script or the `create-plugin update` script did it, don't remember exactly.

Reply:
Hmm... gonna need to take a look at that then. I don't think we should ever tamper with a plugin grafana dependencies unless it's a new scaffold.